packages/core/src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -162,6 +165,42 @@ export class AbstractQueryInterfaceTypeScript {
     }
   }
 
+  /**
+   * Truncates a table
+   *
+   * @param tableName
+   * @param options
+   */
+  async truncateTable(tableName: TableNameOrModel, options?: TruncateTableOptions): Promise<void> {
```

Comment:
Considering Model.truncate exists, I think we should name this `truncate` too

Reply:
Do you want me to update the name of the types as well to `QiTruncateOptions` and `TruncateQueryOptions`?
Or just change `TruncateTableOptions` to `QiTruncateTableOptions`?

Reply:
I've update only the QI name to truncate